lib/Notification/Notifier.php

```diff
@@ -287,6 +293,62 @@ protected function shortenJsonEncodedMultibyteSave(string $subject, int $dataLen
 		return $temp;
 	}
 
+	private function parseStoredRecording(
+		INotification $notification,
+		Room $room,
+		Participant $participant,
+		IL10N $l
+	): INotification {
+		$parameters = $notification->getSubjectParameters();
+		$shareAction = $notification->createAction()
+			->setParsedLabel($l->t('Share to chat'))
+			->setPrimary(true)
+			->setLink(
+				$this->urlGenerator->linkToRouteAbsolute(
+					'ocs.spreed.Chat.shareObjectToChat',
```

Comment:
```suggestion
				$this->urlGenerator->linkToOCSRouteAbsolute(
					'spreed.Chat.shareObjectToChat',
```

Reply:
We could also have `/api/{apiVersion}/recording/{token}/notification` that does this. But both work I guess.